frontend/src/components/Sidebar/sidebar.module.css

```diff
@@ -61,6 +61,19 @@
   font-weight: 700;
 }
 
+.logout {
+  color: #0084F4;
+  position: absolute;
+  text-align: center;
+  bottom: 1.75rem;
+  width: 100%;
+}
+
+.logout>img {
+  width: 50%;
```

Comment:
i think 50% is a little too big, it should match the size of the sidebar circles